src/View/Helper/HtmlHelper.php

```diff
@@ -232,7 +232,11 @@ public function meta($type, $content = null, array $options = [])
             ];
 
             if ($type === 'icon' && $content === null) {
-                $types['icon']['link'] = 'favicon.ico';
```

Comment:
@ADmad Doing it that way breaks the `HtmlHelperTest::testMetaIcon` test. The final expectation in there doesn't match, because the helper returns `href="/testing/testing/favicon.ico"` when changing the code with your suggestion, rather than the expected `/testing/favicon.ico`.